staging/src/k8s.io/api/storage/v1beta1/types.go

```diff
@@ -350,6 +350,21 @@ type CSINodeDriver struct {
 	// This can be empty if driver does not support topology.
 	// +optional
 	TopologyKeys []string `json:"topologyKeys" protobuf:"bytes,3,rep,name=topologyKeys"`
+
+	// allocatable represents the volume resources of a node that are available for scheduling.
+	// +optional
+	Allocatable *VolumeNodeResources `json:"allocatable,omitempty" protobuf:"bytes,4,opt,name=allocatable"`
+}
+
+// VolumeNodeResources is a set of resource limits for scheduling of volumes.
+type VolumeNodeResources struct {
+	// Maximum number of unique volumes managed by the CSI driver that can be used on a node.
+	// A volume that is both attached and mounted on a node is considered to be used once, not twice.
+	// The same rule applies for an unique volume that is shared among multiple pods on the same node.
+	// This field can be nil if there is no limit of volumes supported on a node.
+	// If this field is nil, then the supported number of volumes on this node is unbounded.
```

Comment:
Make sure to sync the comments from all the types.go files

Reply:
Fixed